agent_packager/utils.py

```diff
@@ -120,18 +121,9 @@ def download_file(url, destination):
 
 
 def tar(source, destination):
-    # TODO: solve or depracate..
-    # TODO: apparently, it will tar the first child dir of
```

Comment:
yes well it will do that if you explicitly ask for that by passing `arcname=` that skips the parent.. :D